Libraries/pbxbuild/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ add_library(pbxbuild SHARED
             Sources/DerivedDataHash.cpp
```

Comment:
Yeah, that was the original idea. I'd tried using that to put them in the dependency graph, but projects expect all of them to be written first and then used in custom scripts, etc. So they're already all being written before any invocations run.

In a some cases (scripts, linker file lists, Swift input) they were attached to the invocations that needed them. In a few cases (PCH info) they were totally independent. In others (header maps, module maps) they were dependencies but attached to the wrong invocation because the right one wasn't available.

Fixing the last set to work as dependencies was going to be pretty complicated especially for ones like module maps that are shared between multiple invocations, and not too useful due to the script input issue, so I sorta gave up on the idea.